code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm

```diff
@@ -1,4 +1,4 @@
-#define CHALLENGE_TELECRYSTALS 280
+#define CHALLENGE_RED_TELECRYSTALS 280
```

Comment:
If red telecrystals are used for stealthy, generally-non-murderboney stuff, shouldn't nukies be getting black TC?

Reply:
i'm not clear on whether black can buy both but they should get dark red if it can't